tensorflow/contrib/factorization/BUILD

```diff
@@ -224,8 +224,8 @@ py_test(
     srcs_version = "PY2AND3",
     tags = [
         "manual",
-        "notsan",
-    ],  # b/67512932
+        "notsan",  # b/67512932
```

Comment:
Could you try this again at head? I think it is already fixed. 

https://github.com/tensorflow/tensorflow/blob/2d0531d72c7dcbb0e149cafdd3a16ee8c3ff357a/tensorflow/contrib/factorization/BUILD#L229

Reply:
@zheng-xq I'm cherrypicking this to r1.5 for a patch not master.